app/processing/settlement/schema.js

```diff
@@ -1,7 +1,9 @@
 const Joi = require('joi')
 
 module.exports = Joi.object({
+  invoiceNumber: Joi.string().required(),
   paymentRequestId: Joi.number().integer().required(),
   reference: Joi.string().required(),
-  settled: Joi.boolean().required()
+  settled: Joi.boolean().required(),
+  settlementDate: Joi.date().required()
```

Comment:
Only values we NEED from `settlement` are: `paymentRequestId`, `reference` and `settlementDate` -- remove the other fields